pkg/api/handlers/libpod/containers.go

```diff
@@ -285,3 +285,23 @@ func Restore(w http.ResponseWriter, r *http.Request) {
 	}
 	utils.WriteResponse(w, http.StatusOK, entities.RestoreReport{Id: ctr.ID()})
 }
+
+func InitContainer(w http.ResponseWriter, r *http.Request) {
+	name := utils.GetName(r)
+	runtime := r.Context().Value("runtime").(*libpod.Runtime)
+	ctr, err := runtime.LookupContainer(name)
+	if err != nil {
+		utils.ContainerNotFound(w, name, err)
+		return
+	}
+	err = ctr.Init(r.Context())
+	if errors.Cause(err) == define.ErrCtrStateInvalid {
+		utils.Error(w, "container already initd", http.StatusNotModified, err)
```

Comment:
```suggestion
		utils.Error(w, "container already init'ed", http.StatusNotModified, err)
```
Seems better english.